distribution/docs/src/main/resources/_contents/introductioncontents.adoc

```diff
@@ -0,0 +1,105 @@
+{branding} is a free and open-source common data layer that abstracts services and business logic from the underlying data structures to enable rapid integration of new data sources.
+
+Licensed under http://www.gnu.org/licenses/gpl.html[LGPL], {branding} is an interoperability platform that provides secure and scalable discovery and retrieval from a wide array of disparate sources.
+{branding} is built to "unzip and run" while having the ability to scale to large enterprise systems.
+
+Distributed Data Framework ({branding}) is an agile and modular integration framework.
```

Comment:
Is "agile" the right word here? Feels like we're using it because it's a buzz word. Can we replace that with something like "flexible" or "configurable"?